### PR TITLE
Create ArturMuniz.md

### PR DESCRIPTION
**Pull request by ArturMuniz**

### CODE REVIEW

**Review by github-education — DISMISSED** [review]
**Oie ArturMuniz,**
**Bem vindo a Campus Party!**


### I have a few items I need you to take care of before I can merge this PR:

* *Por favor reveja seu arquivo markdown, possui alguns erros:* 
	*The attribute time need to be in the format: YYYY-MM-DD HH:MM:SS. For example: 2022-06-15 17:01:10*

Feel free to re-request a review from me and I'll come back and take a look!

File: _messages/ArturMuniz.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+
+user: ArturMuniz
+
```
Comment (juanpflores) [inline _messages/ArturMuniz.md:4]:
```suggestion
```

File: _messages/ArturMuniz.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+time: 2022-06-17 1:30
+
+quote: jogar com os amigos minigames, assitir as palestras e participar dos workshops.
+
+---
```
Comment (juanpflores) [inline _messages/ArturMuniz.md:9]:
```suggestion
---
user: ArturMuniz
time: 2022-06-17 01:30
quote: jogar com os amigos minigames, assitir as palestras e participar dos workshops.
---
```

**Review by github-education — CHANGES_REQUESTED** [review]
**Oie ArturMuniz,**
**Bem vindo a Campus Party!**


### I have a few items I need you to take care of before I can merge this PR:

* *Por favor reveja seu arquivo markdown, possui alguns erros:* 
	*The attribute time need to be in the format: YYYY-MM-DD HH:MM:SS. For example: 2022-06-15 17:01:10*

Feel free to re-request a review from me and I'll come back and take a look!

**Review by github-education — CHANGES_REQUESTED** [review]
**Oie ArturMuniz,**
**Bem vindo a Campus Party!**


### I have a few items I need you to take care of before I can merge this PR:

* *Por favor reveja seu arquivo markdown, possui alguns erros:* 
	*The attribute time need to be in the format: YYYY-MM-DD HH:MM:SS. For example: 2022-06-15 17:01:10*

Feel free to re-request a review from me and I'll come back and take a look!

**Review by juanpflores — APPROVED** [review]

**Pull request merged**